app/(gcforms)/[locale]/(app administration)/admin/(with nav)/accounts/[id]/manage-permissions/components/client/Privilege.tsx

```diff
@@ -26,7 +27,10 @@ export const Privilege = ({
           offLabel={t("off")}
           description={description || ""}
           handleToggle={async () => {
-            await updatePrivileges(userId, privilegeId, active ? "remove" : "add");
+            const result = await updatePrivileges(userId, privilegeId, active ? "remove" : "add");
+            if (result?.error) {
+              toast.error(t("error.permissions"), "wide");
```

Comment:
In the future we should decide on a pattern for returning errors from an action. This is a hardcoded error and would break if the server action could return two or more errors.
One idea is to have the server action return the error key.
Future thought :)